src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/CreateSampleResponse.java

```diff
@@ -0,0 +1,25 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto;
+
+/**
+ * UI response to create a new sample
+ */
+public class CreateSampleResponse extends UpdateSampleResponse {
```

Comment:
This probably should have been caught earlier but I think it would be better to just have a single DTO for the `CreateSampleResponse` and `UpdateSampleResponse` which would have 3 constructors. This would allow us to send error messages back if required in the future if there was an error creating a sample. I provided an example below. Let me know what you think! :)

Example: 
```
public class SampleResponse {
  private Long sampleId;
  private boolean error;
  private String errorMessage;
  
  public SampleResponse(Long sampleId) {
    this.sampleId = sampleId;
    this.error = false;
    this.error = null;
  }

  public SampleResponse(Long sampleId, String errorMessage) {
    this.sampleId = sampleId;
    this.error = true;
    this.error = errorMessage;
 }

  public SampleResponse(Long sampleId, String errorMessage) {
    this.sampleId = sampleId;
    this.error = true;
    this.error = errorMessage;
    }

  ....getters and setters here....
}
```

Reply:
I can do this, but CreateSampleResponse already returns errors. I have a constructor that takes error messages.
![image](https://user-images.githubusercontent.com/325703/215164061-5fcb54b7-9bbe-4418-a914-e93e1a68f344.png)

Reply:
If we want to keep it the way it currently is then The `UpdateSampleResponse` should be extending the `CreateSampleResponse` since that matches the flow of the logic (create comes before update). That was my reasoning to create the `SampleResponse` dto which would handle responses for both the create and update. If we update the `UpdateSampleResponse` to extend `CreateSampleResponse` the `UpdateSampleResponse` would be empty. I will leave it upto you as to which approach you would like to take. I'm good with either :)

Reply:
Please see 55c2e2ab2c708e0800fcf71173bdc62f1c93b0b2.

Reply:
Looks great! Thank you!